src/js/actions/search/commands.js

```diff
@@ -60,12 +61,10 @@ define(function (require, exports) {
                 }
 
                 if (menuTree.$MENU) {
-                    resultPath += menuTree.$MENU + ">";
+                    resultPath += menuTree.$MENU + " > ";
```

Comment:
Thanks for the research! We might want to chat about unifying all these separators with @placegraphichere, but it doesn't seem like a major problem right this moment.